traffic_ops/app/db/migrations/20200520000000_add_deleted_tables.sql

```diff
@@ -0,0 +1,470 @@
+/*
```

Comment:
There are some new tables to account for here: https://github.com/apache/trafficcontrol/pull/4687/files